src/main/java/org/folio/services/ResponseHandlerUtil.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.services;
+
+import static javax.ws.rs.core.HttpHeaders.CONTENT_TYPE;
+
+import javax.ws.rs.core.Response;
+
+public final class ResponseHandlerUtil {
+  private static final String HRID_ERROR_MESSAGE = "lower(f_unaccent(jsonb ->> 'hrid'::text))";
+  private static final String HRID = "HRID";
+
+  private ResponseHandlerUtil() {
+  }
+
+  public static Response handleResponse(Response response) {
```

Comment:
It'd be better also to rename this method because it specifically designed to modify istance hrid error response message. See previous messages for a name suggestion